fighthealthinsurance/common_view_logic.py

```
@@ -1646,3 +1651,113 @@ async def format_response(response: dict[str, str]) -> str:
         async for i in interleaved:
             logger.debug(f"Yielding {i}")
             yield i
+
+
+class StripeWebhookHelper:
+    @staticmethod
+    def handle_checkout_session_completed(request, session):
+        try:
+            try:
+                metadata = session.metadata
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)